src/renderer/services/keystore/eth2.ts

```diff
@@ -59,6 +61,11 @@ export class Eth2Keystore implements ICGKeystore {
     }
 
     public getAddress(): string {
+        const words = bech32.toWords(Buffer.from(this.keystore.pubkey, "utf-8"));
+        return bech32.encode(ETH2_ADDRESS_PREFIX, words);
+    }
+
+    public getPublicKey(): string {
         return this.keystore.pubkey.toString();
```

Comment:
isn't this suppose to be .toString("hex")

Reply:
I just figured out that `this.keystore.pubkey` is already a hex string

Reply:
there is not toString method on string. I will return just pubkey, since it is already a string